yt/visualization/plot_window.py

```diff
@@ -2293,14 +2294,63 @@ def _create_axes(self, axrect):
         self.axes = self.figure.add_axes(axrect, projection=self._projection)
 
 
-def SlicePlot(ds, normal=None, fields=None, axis=None, *args, **kwargs):
+def _sanitize_normal_vector(ds, normal) -> Union[str, np.ndarray]:
+    """Return the name of a cartesian axis whener possible,
+    or a 3-element 1D ndarray of float64 in any other valid case.
+    Fail with a descriptive error message otherwise.
+    """
+    axis_names = ds.coordinates.axis_order
+
+    if isinstance(normal, str):
+        if normal not in axis_names:
+            raise ValueError(
+                f"'{normal}' is not a valid axis name. " "Expected either ."
```

Comment:
I think there's a bit missing at the end, isn't there?

```suggestion
                f"{normal} is not a valid axis name. Expected either ."
```

Reply:
Oh yeah right I must have forgotten I was writing this, thanks !